python/py-setuptools/Portfile

```diff
@@ -29,29 +29,29 @@ checksums           md5 5be48906d7c00b198da720ab45cff26a \
                     rmd160 d5421551ef79d6a5cb43b38cd619590d04857697 \
                     sha256 39060a59d91cf5cf403fa3bacbb52df4205a8c3585e0b9ba4b30e0e19d4c4b18
 
-python.versions     26 27 33 34 35 36 37 38
+python.versions     26 27 33 34 35 36 37 38 pypy36 pypy37
 python.link_binaries no
 
 if {$subport ne $name} {
-    if {${python.version} == 27} {
+    if {${python.branch} eq "2.7"} {
```

Comment:
Having to make these kinds of changes in all the ports that use python.version like this is problematic.

Reply:
Yeah, I reverted the change such that `${python.version}` and `${python.branch}` retain their values whether targeting PyPy or CPython.